packages/amplify-cli/src/attach-backend.ts

```diff
@@ -59,7 +59,9 @@ async function onSuccess(context: $TSContext) {
   await postPullCodegen(context);
 
   if (!inputParams.yes) {
-    const confirmKeepCodebase = await context.amplify.confirmPrompt('Do you plan on modifying this backend?', true);
+    const confirmKeepCodebase = context.exeInfo.existingLocalEnvInfo?.noUpdateBackend
```

Comment:
Can this logic be moved into a function, like checkBackendUpdated() or something to that effect?